src/Symfony/Component/Console/Tester/CommandTester.php

```diff
@@ -129,4 +135,29 @@ public function getStatusCode()
     {
         return $this->statusCode;
     }
+
+    /**
+     * Sets the user inputs.
+     *
+     * @param array An array of strings representing each input
+     *              passed to the command input stream.
+     *
+     * @return CommandTester
+     */
+    public function setInputs(array $inputs)
+    {
+        $this->inputs = $inputs;
+
+        return $this;
+    }
+
+    private static function createStream(array $inputs)
```

Comment:
No, I meant public. I think it should be private, I don't see why an end user would have to use this.